lib/cli/driver.js

```diff
@@ -75,15 +75,20 @@ class DriverCommand {
       if (!showUpdates) {
         return;
       }
-      for (const [, data] of _.toPairs(drivers)) {
-        const {version, pkgName, installType} = data;
-        data.updateVersion = null;
-        if (installType === INSTALL_TYPE_NPM) {
-          const latestVersion = await this.npm.getLatestVersion(pkgName);
-          if (!version || util.compareVersions(latestVersion, '>', version)) {
-            data.updateVersion = latestVersion;
-          }
+      for (const [driver, data] of _.toPairs(drivers)) {
+        const {installed, installType} = data;
+        if (!installed) {
+          // don't need to check for updates on drivers that aren't installed
+          continue;
+        }
+        if (installType !== INSTALL_TYPE_NPM) {
+          // don't need to check for updates on non-npm drivers
```

Comment:
why so? if a driver is installed from github then there might be repository updates since last pull

Reply:
we install drivers from github using npm anyway, and i don't think there's an easy way to check if there are updates. to me, the non-npm installation approaches are 'advanced' and i don't want to build a lot of infrastructure around updating them. an update for them is also trivial in any case:

```
appium driver uninstall foo && appium driver install --source git git+https://github.com/foo/foo
```